docs/preview/features/message-pumps/service-bus.md

```diff
@@ -167,6 +167,10 @@ public class Startup
                 // this job instance in a multi-instance deployment (default: guid).
                 options.JobId = Guid.NewGuid().ToString();
 
+                // The name of the Azure Service Bus message property that has the transaction ID.
+                // (default: Transaction-Id).
+                options.TransactionIdPropertyName = "X-Transaction-ID";
```

Comment:
What about this?
```suggestion
                options.Correlation.TransactionIdPropertyName = "X-Transaction-ID";
```

Because I'm also considering adding a flag where a new transaction id is generated in case it's not specified on the message. Thoughts on that?

Reply:
Yes, that seems like a good approach, similar as we did in the web API repo. I'll change that!

Reply:
Updated this. Come and see! 😉